about.html

```diff
@@ -0,0 +1,210 @@
+        <nav class="nav flex-row w-100">
+          <svg
+            id="btn"
+            class="nav_botton hamb hamRotate ham"
+            viewBox="0 0 100 100"
+            width="60"
+          >
+            <path
+              class="line top"
+              d="m 30,33 h 40 c 3.722839,0 7.5,3.126468 7.5,8.578427 0,5.451959 -2.727029,8.421573 -7.5,8.421573 h -20"
+            />
+            <path class="line middle" d="m 30,50 h 40" />
+            <path
+              class="line bottom"
+              d="m 70,67 h -40 c 0,0 -7.5,-0.802118 -7.5,-8.365747 0,-7.563629 7.5,-8.634253 7.5,-8.634253 h 20"
+            />
+          </svg>
+          <div class="logo-wrapper">
+            <a href="/"
+              ><img
+                width="150px"
+                height="auto"
+                src="./image/logo.png"
+                alt="logo"
+            /></a>
+          </div>
+          <div class="menu-container" id="menu-show">
+            <span id="menu-close" class="menu-close"
+              ><nav class="close-icon">
+                <img src="./image/Close-Icon.png" alt="close-icon" /></nav
+            ></span>
+
+            <div class="menu-list">
+              <ul class="nav nav-list-wrapper flex-row">
+                <li class="nav-item">
+                  <a class="menu-item" href="/index.html">About</a>
+                </li>
+                <li class="nav-item">
+                  <a class="menu-item" href="#program">Program</a>
+                </li>
+                <li class="nav-item">
+                  <a class="menu-item" href="#about">join</a>
+                </li>
+                <li class="nav-item">
+                  <a class="menu-item" href="#about">sponsor</a>
+                </li>
+                <li class="nav-item">
+                  <a class="menu-item" href="#logo-section">News</a>
+                </li>
+              </ul>
+              <div class="canpion">
+                <div class="canpion">
+                </div>
+              </div>
+            </div>
+          </div>
+        </nav>
+
+        <!-- deskop-version -->
+        <div class="deskop-version">
+          <nav class="navbar">
+            <ul class="window-desktop-nav">
+              <li class="nav-item1">
+                <a class="menu-item1" href="#about">About</a>
+              </li>
+              <li class="nav-item1">
+                <a class="menu-item1" href="#program">Program</a>
+              </li>
+              <li class="nav-item1">
+                <a class="menu-item1" href="#join">Join</a>
+              </li>
+              <li class="nav-item1">
+                <a class="menu-item1" href="#">Sponsor</a>
+              </li>
+              <li class="nav-item1">
+                <a class="menu-item1" href="#logo-section">News</a>
+              </li>
+              <li class="canpion">
+                <a class="menu-item1" href="/index.html"><h2>WL Summit</h2></a>
+              </li>
```

Comment:
- [OPTIONAL] You can also fix your menu for the desktop of the About page by adding the link to the Home page. As I mentioned before it's not every user can know if the logo redirects to the home page. So you can add the link to the Home page so the user can navigate between the two pages easily 😁 😁 
- Please also fix your design 🙃, the logo is supposed to be to the left, and the nav-link to the right. But your menu all are on the left side 😁 😁 

Example of how your menu can look on about page

<img width="1004" alt="Screen Shot 2023-01-14 at 22 08 36" src="https://user-images.githubusercontent.com/74501523/212492336-345eb59e-c375-4519-90e4-547c7f50d04e.png">

Example of code:

<img width="597" alt="Screen Shot 2023-01-14 at 22 09 34" src="https://user-images.githubusercontent.com/74501523/212492368-82de8f75-3de0-4635-92b7-71bc1bf967f0.png">